cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -657,8 +657,8 @@ to make sure the CDAP table definitions in HBase are up-to-date.
 These steps will stop CDAP, update the installation, run an upgrade tool for the table definitions,
 and then restart CDAP.
 
-These steps will upgrade from CDAP 2.8.0 to 3.0.0. (**Note:** Apps need to be both
-recompiled and re-deployed.) 
+These steps will upgrade from CDAP 3.0.3 to 3.1.0. (**Note:** Some apps need to be both
+recompiled and re-deployed, see below.)
```

Comment:
These steps will upgrade from CDAP 3.0.3 to 3.1.0. If you are on an earlier version of CDAP, please follow the upgrade instructions for the earlier versions and upgrade first to 3.0.3 before proceeding.